advanced/helm/integrator-with-analytics/integrator-with-analytics/templates/integrator-analytics-deployment.yaml

```diff
@@ -12,31 +12,56 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: integrator-with-analytics-ei-analytics-deployment
-  namespace : {{ .Values.namespace }}
+  namespace : {{ .Release.Namespace }}
 spec:
-  replicas: 2
-  minReadySeconds: 30
+  replicas: {{ default 2 .Values.wso2.deployment.wso2eiAnalyticsWorker.replicas }}
+  minReadySeconds: {{ default 2 .Values.wso2.deployment.wso2eiAnalyticsWorker.minReadySeconds }}
   strategy:
     rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 0
+      maxSurge: {{ default 1 .Values.wso2.deployment.wso2eiAnalyticsWorker.strategy.rollingUpdate.maxSurge }}
+      maxUnavailable: {{ default 0 .Values.wso2.deployment.wso2eiAnalyticsWorker.strategy.rollingUpdate.maxUnavailable }}
     type: RollingUpdate
+  selector:
+    matchLabels:
+      deployment: integrator-with-analytics-ei-analytics
   template:
     metadata:
       labels:
         deployment: integrator-with-analytics-ei-analytics
     spec:
+      initContainers:
+      - name: init-is-db
```

Comment:
@Savidude the InitContainer name needs to be changed to a suitable name such as, `init-ei-db`.

Reply:
Fixed https://github.com/wso2/kubernetes-ei/pull/180/commits/4eab0894505a022e409f632e44196a919575ceb1